crates/expressions/Cargo.toml

```diff
@@ -12,5 +12,6 @@ spin-locked-app = { path = "../locked-app" }
 thiserror = { workspace = true }
 
 [dev-dependencies]
+spin-world = { path = "../world" }
```

Comment:
Is this dependency still needed?

Reply:
Yes, here:

https://github.com/seun-ja/spin/blob/9c7c74285555f6c23834ad3e21564a9633764c7a/crates/expressions/tests/validation_test.rs#L195

Reply:
But the `Provider` trait that you're implementing uses `async_trait` directly rather than the `spin-world` re-export: https://github.com/spinframework/spin/blob/main/crates/expressions/src/provider.rs.  Are you _sure_ you need the `spin-world` re-export?

(There is also a re-export from `spin-locked-app` which is already a dependency.)